### ISSUE TITLE
Reading Scalers

### ISSUE THREAD

**Issue by dfujim** [dfujim/bdata#2]
On occasion, scalar reading appears to chew up memory and does not read properly.

Errors on user cancel (no errors produced during runtime): 

```
  File "/home/bnmr/src/bfit/gui/fileviewer_tab.py", line 166, in get_data
    data = bdata(run,year=year)
  File "/home/bnmr/.local/lib/python3.6/site-packages/bdata/bdata.py", line 468, in __init__
    self.scaler_list[-1].title = str(mp.get_scaler_label(fh,i))
```

```
  File "/home/bnmr/src/bfit/gui/fileviewer_tab.py", line 166, in get_data
    data = bdata(run,year=year)
  File "/home/bnmr/.local/lib/python3.6/site-packages/bdata/bdata.py", line 465, in __init__
    self.scaler_list[-1].counts = mp.get_scaler_counts(fh,i)
```

Seems to be more prominent when running from bdata on muesli.

**Comment by dfujim** [comment]
For now scalar reading is disabled (commented out in commit 5941f9fdba).

**Comment by dfujim** [comment]
Issue was that reading number of scalers failed. Resolved in d14fe93dbe4b2f6905bbe1b8e389637cfc0035ce due to catching of fail states.
